zebrad/tests/acceptance.rs

```diff
@@ -197,7 +207,7 @@ fn seed_args() -> Result<()> {
 #[test]
 fn start_no_args() -> Result<()> {
     zebra_test::init();
-    let (tempdir, _guard) = tempdir(true)?;
+    let (tempdir, _guard) = tempdir(ConfigMode::Ephemeral)?;
```

Comment:
@vramana I just read your comment on Discord, and I understand why we were using persistent here. I'll just change it back.

Sorry about that!

```suggestion
    // start actually uses the state directory, so run one of the start tests with persistent
    let (tempdir, _guard) = tempdir(ConfigMode::Persistent)?;
```